recipes/harfbuzz/all/conanfile.py

```diff
@@ -74,10 +73,15 @@ def configure(self):
         if self.options.shared and self.options.with_glib:
             wildcard = "" if Version(conan_version) < "2.0.0" else "/*"
             self.options[f"glib{wildcard}"].shared = True
+        if not self.options.with_glib:
+            del self.options.with_introspection
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
+    def package_id(self):
+        self.info.options.with_subset = True
```

Comment:
Why not `del` this option altogether though? Keeping the old value around is meaningless as the new and old versions are both getting the new `with_introspection` option so the packageids will be different no?

Reply:
No real reason, just forgot that I could do that. I'll fix that.